bin/src/runRollup.js

```diff
@@ -217,7 +216,7 @@ function bundle ( options ) {
 		handleError({ code: 'MISSING_INPUT_OPTION' });
 	}
 
-	return rollup.rollup( options ).then( bundle => {
+	return rollup( options ).then( bundle => {
```

Comment:
@Rich-Harris more like the Christmas Hat Cat. I love that guy. SO unfortunate about his neck tho. https://www.youtube.com/watch?v=159ERq6Kf9Y

/cc @GGAlanSmithee